kernel/cClosure.ml

```diff
@@ -1667,6 +1698,7 @@ let whd_val info tab v = term_of_fconstr (kh info tab v [])
 
 (* strong reduction *)
 let norm_val info tab v = kl info tab v
+let norm_term info tab e t = klt info tab e t
```

Comment:
This may be a nicer api
```suggestion
let norm_term info tab t = klt info tab (Esubst.subs_id 0, Univ.Instance.empty) t
```

Reply:
The other one is more general, though. (In any case I think this should be moved out of CClosure, the only use in the kernel is just for user-printing errors so in the end there is only a single caller in Reductionops, where this function can be conveniently made internal.)